test/inductor/test_cuda_repro.py

```diff
@@ -928,6 +928,14 @@ def fn(arg7_1, add_1, permute_2, select_scatter, slice_8):
         ref = compiled(list(args))
         assert same(ref, correct)
 
+    def test_issue104937(self):
```

Comment:
Suggest you provide a meaningful name for the function. You may add an issue link in the comment.